webplugin/js/app/zendesk-chat-service.js

```diff
@@ -247,7 +249,27 @@ function ZendeskChatService() {
             },
         });
 
-    }
+    };
+
+    _this.handleZendeskAgentLeaveEvent = function (event) {
+        //Resolve conversation on widget
+        KommunicateUI.showClosedConversationBanner(
+            true
+        );
+        KommunicateUI.isConvJustResolved = true;
+        
+        //Call API to resolve the conversation on Dashboard
+        kommunicate.client.resolveConversation({ 
+            groupId: CURRENT_GROUP_DATA.tabId 
+        }, function(err, result) {
```

Comment:
i guess we are supposed to show some new text "Start Chat Again", and on click of that it should create a new conversation. can you check once with Shweta to verify if current functionality is fine or not.

Reply:
Few changes were required. Added them.